src/Servers/Kestrel/Core/test/Http1/Http1ConnectionTests.cs

```diff
@@ -696,6 +696,38 @@ public async Task RequestAbortedTokenIsResetBeforeLastWriteAsyncWithContentLengt
         Assert.False(_http1Connection.RequestAborted.IsCancellationRequested);
     }
 
+    [Fact]
+    public async void BodyWriter_OnAbortedConnection_ReturnsFlushResultWithIsCompletedTrue()
+    {
+        var payload = Encoding.UTF8.GetBytes("hello, web browser" + new string(' ', 512) + "\n");
+        var writer = _application.Output;
+
+        var successResult= await writer.WriteAsync(payload);
+        Assert.False(successResult.IsCompleted);
+
+        _http1Connection.Abort(new ConnectionAbortedException());
+        var failResult= await _http1Connection.FlushPipeAsync(new CancellationToken());
+        Assert.True(failResult.IsCompleted);
+    }
```

Comment:
Nit:

```suggestion
    }

```

Can you also fix the formatting of the `=` signs in `var failResult= await _http1Connection.FlushPipeAsync(new CancellationToken());` and elsewhere?